docs/02.API-REFERENCE.md

```diff
@@ -101,6 +101,24 @@ created by API functions has the error flag set.
 typedef uint32_t jerry_value_t;
 ```
 
+## jerry_context_data_manager_t
+
+**Summary**
+
+Structure that defines how a user context item will be initialized and deinitialized. The default initialization, if
+`init_cb` is NULL, is to zero out the memory.
+
+**Prototype**
+
+```c
+typedef struct
+{
+  void (*init_cb) (void *); /**< callback responsible for initializing a context item, or NULL to zero out the memory */
```

Comment:
It actually is always zeroed, but that's not how I documented it. I'll re-word this to reflect that.